src/main/java/sg/ncl/MainController.java

```diff
@@ -132,9 +149,26 @@ public void futureplanDownload(HttpServletResponse response) {
             IOUtils.copy(is, response.getOutputStream());
             response.flushBuffer();
             is.close();
+        } catch (Exception ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)